src/components/Cards/PlayedCards/PlayedCards.jsx

```diff
@@ -4,9 +4,10 @@ import styles from '../Cards.module.css';
 
 export class PlayedCards extends React.Component {
     render() {
+        console.log(this.props.cards);
         return (
             <ul id="played-cards">
-                {this.props.cards.map(card => <img id={"card-" + card.id} className={styles.allCards} alt={"card-" + card.id} key={card.id} src={require(`../cards/card (${card.id}).jpg`)}/>)}
+                {this.props.cards.map(card => <img id={"card-" + card.cardId} key={card.cardId} className={styles.allCards} alt={"card-" + card.cardId} key={card.id} src={require(`../cards/card (${card.cardId}).jpg`)}/>)}
```

Comment:
I could be wrong but I don't think you actually need to `require` the image file here. You might be able to just use relative paths.